deployment/terraform/cdn.tf

```diff
@@ -85,7 +85,7 @@ resource "aws_cloudfront_distribution" "cdn" {
 
   logging_config {
     include_cookies = false
-    bucket          = "${aws_s3_bucket.logs.id}.s3.amazonaws.com"
+    bucket          = "${aws_s3_bucket.logs.bucket_domain_name}"
```

Comment:
These two create no diff in the plan, but using `bucket_domain_name` requires less manual stringiness.